### PR TITLE
refactor(GiniHealthAPILibrary): Remove pinning projects

### PR DESCRIPTION
**Pull request by antosj**

### CODE REVIEW

File: .github/workflows/health-sdk.check.yml [hunk 1]
```diff
@@ -36,43 +36,14 @@ jobs:
...
     - name: Run unit tests
       run: |
         xcodebuild clean test -workspace GiniMobile.xcworkspace -scheme "${{ matrix.target }}Tests" -destination "${{ matrix.destination }}" CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO ONLY_ACTIVE_ARCH=NO
-   
+
     - name: Build example app
       run: |
-        xcodebuild -project HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample.xcodeproj -scheme "GiniHealthSDKPinningExample" -destination "${{ matrix.destination }}" CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO ONLY_ACTIVE_ARCH=NO    
-
-    - name: Build pinning example app and run integration tests
```
Comment (zladzeyka) [inline .github/workflows/health-sdk.check.yml:56]: [Suggestion]: @antosj we need to run pinning-related tests just using the GiniHealthSDKExampleTests scheme

**Review by zladzeyka — CHANGES_REQUESTED** [review]
@antosj looks great just 2 tiny suggestions from my side

File: .github/workflows/health-api-library.check.yml [hunk 2]
```diff
@@ -62,23 +57,13 @@ jobs:
     - name: Build pinning example app and run integration tests
       run: >
         xcodebuild clean test
-        -project HealthAPILibrary/GiniHealthAPILibraryPinningExample/GiniHealthAPILibraryPinningExample.xcodeproj
-        -scheme "GiniHealthAPILibraryPinningExampleTests"
+        -project HealthAPILibrary/GiniHealthAPILibraryExample/GiniHealthAPILibraryExample.xcodeproj
+        -scheme "GiniHealthAPILibraryTests"
         -destination "${{ matrix.destination }}"
         -skip-testing:GiniHealthAPILibraryPinningExampleTests/HealthAPILibraryPinningWrongCertificatesTests
```
Comment (zladzeyka) [inline .github/workflows/health-api-library.check.yml:63]:
```suggestion
        -skip-testing:GiniHealthAPILibraryExampleTests/HealthAPILibraryPinningWrongCertificatesTests
```

Please check what are you running here

File: .github/workflows/merchant-sdk.check.yml [hunk 3]
```diff
@@ -68,4 +68,3 @@ jobs:
         CODE_SIGNING_REQUIRED=NO
         ONLY_ACTIVE_ARCH=NO
         CLIENT_ID="gini-mobile-test"
-        CLIENT_SECRET="${{ secrets.GINI_MOBILE_TEST_CLIENT_SECRET }}"
```
Comment (zladzeyka) [inline .github/workflows/merchant-sdk.check.yml:71]: Please revert this change

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Thank you @antosj 
please check my 2 comments below

File: .github/workflows/health-sdk.publish.example.apps.yml [hunk 4]
```diff
@@ -7,7 +7,7 @@ on:
   workflow_dispatch:
 jobs:
   check:
-    uses: gini/gini-mobile-ios/.github/workflows/health-sdk.check.yml@main
+    uses: gini/gini-mobile-ios/.github/workflows/health-sdk.check.yml
```
Comment (zladzeyka) [inline .github/workflows/health-sdk.publish.example.apps.yml]:
```suggestion
    uses: ./.github/workflows/health-sdk.check.yml
```

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Only 1 tiny suggestion

**Review by razvancapra — APPROVED** [review]

**Review by zladzeyka — APPROVED** [review]

**Pull request merged**